index-fetch.js

```diff
@@ -1,5 +1,9 @@
 'use strict'
 
+const Dispatcher = require('./lib/dispatcher/dispatcher')
+const api = require('./lib/api')
+Object.assign(Dispatcher.prototype, api)
+
```

Comment:
It's required if it's the default dispatcher. Any ProxyAgent as the default dispatcher causes many tests to fail without this.

I'll go ahead and remove though since it won't be default.